aws_analysis_tools/cli/update_ec2_tags.py

```diff
@@ -43,120 +41,134 @@ def add_cli_arguments(self, parser):
 
         group.add_argument(
             '--instance-id',
-            help = "The instance id of the instance. Example: 'i-27459bcc' "
-            "(output of `facter ec2_instance_id`)"
+            help=("The instance id of the instance. Example: 'i-27459bcc' "
+                  "(output of `facter ec2_instance_id`)")
         )
 
         group.add_argument(
             '--ec2-region',
-            required = True,
-            help     = "EC2 region to use. Example: 'us-east-1'. "
-            "NB: This is the *region*, not the *availability zone*."
+            required=True,
+            help=("EC2 region to use. Example: 'us-east-1'. "
+                  "NB: This is the *region*, not the *availability zone*.")
         )
 
         group.add_argument(
             '--environment',
-            default = 'dev',
-            help    = "The environment to set. (default: %(default)s)."
+            default='dev',
+            help="The environment to set. (default: %(default)s)."
         )
 
         group.add_argument(
             '--cluster-name',
-            required = True,
-            help     = "The cluster name to set. Example: 'apiservices-a'"
+            required=True,
+            help="The cluster name to set. Example: 'apiservices-a'"
         )
 
         group.add_argument(
             '--classes',
-            nargs   = "*",
-            default = [],
-            help    = "Specify class to tag with (can be used multiple times). "
-            "(default: %(default)s)"
+            nargs="*",
+            default=[],
+            help=("Specify class to tag with (can be used multiple times). "
+                  "(default: %(default)s)")
+        )
+
+        group.add_argument(
+            '--release',
+            # Let's make sure to tag this for now
+            # TODO: Pull out the code to deduce this from krux-manage-instance and use it here.
+            required=True,
+            help=("The name of the ubuntu release to set. Example: Trusty"),
```

Comment:
The instance will be tagged with whatever you pass. The use case for this application is to run it in puppet and passing facter's `lsbdistcodename`.

If we want, I can set `choice` option to make sure the correct option is passed. However, that means this application will break when we start to use 16.04 or 18.04 in future.